.github/workflows/release.yaml

```diff
@@ -64,10 +61,10 @@ jobs:
       - name: Release
         uses: goreleaser/goreleaser-action@v2
         with:
-          version: v0.183.0
+          version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.ORG_REPO_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We cannot use the default token since this action pushes commits to another repository. If you need additional permission, please let me know.

Reply:
so sorry about that, I forgot to revert the changes that I've been made for testing purposes.